examples/01_plotting/plot_visualize_megatrawls_netmats.py

```diff
@@ -28,10 +28,9 @@
 # plotting correlation matrices
 import matplotlib.pyplot as plt
 from nilearn import plotting
+from nilearn.plotting.matrix_plotting import plot_matrix
```

Comment:
We should import directly from nilearn.plotting: we strive to avoid deep imports.